server/server.go

```diff
@@ -1471,13 +1477,14 @@ func (s *StanServer) configureNATSServerTLS(opts *server.Options) error {
 
 // startNATSServer massages options as necessary, and starts the embedded
```

Comment:
typo messages i think

Reply:
It may be bad wording, but I think it meant that the nats options are changed if needed in order to start the embedded NATS Server.

Reply:
Ok thx suggest `startNATSServer change options if necessary,`

Reply:
What about:
```
// startNATSServer starts the embedded NATS server, possibly updating
// the NATS Server's clustering and/or TLS options.
```
or remove altogether ;-)